iris_grib/_load_convert.py

```diff
@@ -9,9 +9,6 @@
 
 """
 
-from __future__ import (absolute_import, division, print_function)
-from six.moves import (filter, input, map, range, zip)  # noqa
-
 from argparse import Namespace
 from collections import namedtuple, Iterable, OrderedDict
```

Comment:
I think some of these have "moved" :  Iterable should now be imported from `collections.abc`.
(`namedtuple` and `OrderedDict` are ok)

I'm not sure why this doesn't give a deprecation warning in the Travis log : maybe because it is not "strictly" a Python3 migration issue as it was supposedly "better" to do it this way before ??

According to [this note](https://docs.python.org/3/library/collections.html), this affects _all_ the ABCs like Callable, Iterable etc, and is "Deprecated since version 3.3, will be removed in version 3.9".

I will check for these + raise separate comments elsewhere if I find any ...